test/server_test.py

```diff
@@ -235,6 +254,23 @@ def started_imageio(tmpdir, drop_privileges="true"):
         proc.wait()
 
 
+@requires_root
+def test_load_ticket(tmpdir, tmp_dirs, tmp_image, ticket):
+    data = b"a" * 512 + b"b" * 512
+    tmp_image.write(data)
+    with started_imageio(tmpdir, drop_privileges="false", ticket=ticket):
+        conf_dir = tmpdir.join("conf")
+        cfg = config.load(str(conf_dir.join("conf.d", "daemon.conf")))
+        with http.ControlClient(cfg) as c:
+            res = c.get('/tickets/file')
+            assert res.status == http_client.OK
```

Comment:
We don't need the control service to check that we have a ticket. Do upload/download
using this ticket instead.

The typical configuration when using initial ticket is to disable the control service since in this mode we serve exactly one ticket - so we don't have a control service to check the ticket.